src/java/org/tensorics/core/tensor/options/ExactShapesOrOneZeroStrategy.java

```diff
@@ -34,6 +34,10 @@
  */
 public class ExactShapesOrOneZeroStrategy implements ShapingStrategy {
 
+    public static ExactShapesOrOneZeroStrategy getInstance() {
```

Comment:
If you use a factory method, then I would suggest to call it create(). This is usually the convention used e.g. in guava. getInstance() looks more like the (ugly) singleton pattern....
If we do it here, probably we should do it for all strategies...